app/assets/v2/js/status.js

```diff
@@ -22,6 +23,37 @@ $(document).ready(function() {
   // dropdown for usernames when @ is detected in the post
   $('#textarea').on('input', function(e) {
     const lastWord = e.target.value.split(' ').pop();
+    const youtube_re = /^.*((youtu.be\/)|(v\/)|(\/u\/\w\/)|(embed\/)|(watch\?))\??v?=?([^#\&\?]*).*/;
```

Comment:
This looks like it only matches shortened YouTube URLs, i.e. youtu.be/dak3sl5hf32, and it only matches the beginning of the string. I found one that is more comprehensive and matches all possible combinations of YouTube's URLs:
```suggestion
    const youtube_re = /(?:https?://|//)?(?:www\.|m\.)?(?:youtu\.be/|youtube\.com/(?:embed/|v/|watch\?v=|watch\?.+&v=))([\w-]{11})(?![\w-])/;
```

Reply:
Thanks, i appreciate your feedback! The screenshot shows the short and large URLs are working. I'll try your recommendations 🙂

Reply:
I'll substitute the regex in favor of the proposed by @KiChjang because of improving readability. :muscle: